qiita_pet/handlers/api_proxy/tests/test_studies.py

```diff
@@ -52,28 +52,79 @@ def test_study_get_req(self):
                     'Future studies will attempt to analyze the soils and '
                     'rhizospheres from the same location at different time '
                     'points in the plant lifecycle.',
-                    'status': 'private',
-                    'spatial_series': False,
-                    'study_description': 'Analysis of the Cannabis Plant '
-                                         'Microbiome',
-                    'shared_with': ['shared@foo.bar'],
-                    'lab_person': {'affiliation': 'knight lab',
-                                   'name': 'LabDude',
-                                   'email': 'lab_dude@foo.bar'},
-                    'principal_investigator': {'affiliation': 'Wash U',
-                                               'name': 'PIDude',
-                                               'email': 'PI_dude@foo.bar'},
-                    'study_alias': 'Cannabis Soils',
-                    'study_id': 1,
-                    'most_recent_contact': datetime(2014, 5, 19, 16, 11),
-                    'publications': [['10.100/123456', '123456'],
-                                     ['10.100/7891011', '7891011']],
-                    'num_samples': 27,
-                    'study_title': 'Identification of the Microbiomes for '
-                                   'Cannabis Soils',
-                    'number_samples_collected': 27}}
+                'status': 'private',
+                'spatial_series': False,
+                'study_description': 'Analysis of the Cannabis Plant '
+                                     'Microbiome',
+                'shared_with': ['shared@foo.bar'],
+                'lab_person': {'affiliation': 'knight lab',
+                               'name': 'LabDude',
+                               'email': 'lab_dude@foo.bar'},
+                'principal_investigator': {'affiliation': 'Wash U',
+                                           'name': 'PIDude',
+                                           'email': 'PI_dude@foo.bar'},
+                'study_alias': 'Cannabis Soils',
+                'study_id': 1,
+                'most_recent_contact': datetime(2014, 5, 19, 16, 11),
+                'publications': [['10.100/123456', '123456'],
+                                 ['10.100/7891011', '7891011']],
+                'num_samples': 27,
+                'study_title': 'Identification of the Microbiomes for '
+                               'Cannabis Soils',
+                'number_samples_collected': 27}}
         self.assertEqual(obs, exp)
 
+        # Test with no lab person
```

Comment:
I think this is a good thing to have because PI denotes lab submitting, lab
personnel maybe may be changed to 'submitter' who uploaded the data.  This
allows us to identify source of data (PI) and the person who uploaded the
study.  Often uploader moves to new lab (new position, where they may be
PI) and continues to use qiita.  I believe it is good to track this
history.  It also has an implication for seq_center, center_name because it
will allow us to solve problems with processing.
Do not remove this requirement - it is very easy to add PI even if they
have no account

On Thu, Mar 24, 2016 at 8:54 PM, Jose Navas <notifications@github.com>
wrote:

> In qiita_pet/handlers/api_proxy/tests/test_studies.py
> <https://github.com/biocore/qiita/pull/1708#discussion_r57416936>:
>
> >          self.assertEqual(obs, exp)
> >
> > +        # Test with no lab person
>
> From here below is the new test that I've added.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/qiita/pull/1708/files/a6067ee7ec0f54b4c9572a5fe16bc862b45818b7#r57416936>
>



-- 
Gail Ackermann
Knight Lab
UCSD
glackermann@ucsd.edu <ackermag@ucsd.edu>